apps/communications/contacts/js/contacts.js

```diff
@@ -520,19 +561,19 @@ var Contacts = (function() {
     }
   };
 
+  var handleSelectTagDoneNewDesign = function handleSelectTagDoneNewDesign(){
```

Comment:
you don't need separate function here, since you are just replacing original ``handleSelectTagDone``